src/commands/Tripulacion/tripulacion.js

```diff
@@ -0,0 +1,605 @@
+		const newRolePosition = roleBelow ? (roleBelow.position + 1) : 1;
+
+		const role = await msg.guild.roles.create({ data: {
+			name: crewName,
+			color: roleColor,
+			position: newRolePosition,
+			hoist: true,
+			mentionable: true
+		}});
+
+		await member.roles.add(role);
+
+		// Create crew channels
+		const textChannel = await msg.guild.channels.create(crewName, {
+			type: 'text',
+			parent: channelParentText,
+			overwrites: [{ id: msg.guild.id, denied: ['VIEW_CHANNEL'] }, { id: role.id, allowed: ['VIEW_CHANNEL'] }]
+		});
+
+		const voiceChannel = await msg.guild.channels.create(crewName, {
+			type: 'voice',
+			parent: channelParentVoice,
+			userLimit: 10,
+			overwrites: [{ id: msg.guild.id, denied: ['CONNECT'] }, { id: role.id, allowed: ['CONNECT'] }]
+		});
+
+		// Create the crew in the database, and store initial data
+		const crew = crews.get(crewName, true);
+		if (crew._syncStatus) await crew._syncStatus;
+		await crew.update('role', role);
+		await crew.update('channelText', textChannel);
+		await crew.update('channelVoice', voiceChannel);
+		await crew.update('owner', member);
+		await crew.update('members', member);
+
+		// Success!
+		return msg.sendMessage(`¡Listo! ¡Parece que ${member} sabe nadar y distinguir cuál es el grog bueno! Desde hoy, ¡${member} capitaneará ${role}! ¡Mucha suerte surcando los mares!`);
+	}
+
+	async moderar(msg, [crewName]) {
+		// This command is for admins only, check permission level 6
+		const { permission } = await this.client.permissionLevels.run(msg, 6);
+		if (!permission)
+			throw msg.language.get('INHIBITOR_PERMISSIONS');
+
+		// Gather general crew config vars
+		const { channelReportAdmin, emojiAccept, emojiReject } = msg.guild.configs.crews;
+		const crews = this.client.gateways.crews;
+		await crews.sync();
+
+		// Confirm there's an admin channel for interaction
+		const reportChannel = channelReportAdmin ? msg.guild.channels.find(channel => channel.id === channelReportAdmin) : undefined;
+		if (!reportChannel)
+			throw '¡Maldición! ¿Por qué no crea un canal de administración donde podamos hablar en privado, señor?';
+
+		// List mode
+		if (!crewName) {
+			// Get sorted list of crews with pending changes
+			const pendingCrews = crews.cache.filter(crew => crew.pendingCrewLogo || crew.pendingCrewDescription || crew.pendingCrewImage).sort();
+
+			// Early out if there aren't crews with pending changes
+			if (pendingCrews.size <= 0)
+				throw "¡Salud! No hay cambios pendientes de revisar...";
+
+			const embed = new MessageEmbed()
+				.setTitle('⚓ TRIPULACIONES MÍTICAS - CAMBIOS PENDIENTES')
+				.setColor(0x2b9d98)
+				.setThumbnail("https://i.imgur.com/gVs4OQ5.png");
+
+			pendingCrews.forEach(crew => {
+				let pending = [];
+				if (crew.pendingCrewLogo) pending.push("Logo");
+				if (crew.pendingCrewDescription) pending.push("Historia");
+				if (crew.pendingCrewImage) pending.push("Imagen");
+				pending = pending.join(" + ");
```

Comment:
En vez de cambiar el tipo de la variable `pending`, haz el join en addField

```suggestion
				embed.addField(crew.id, pending.join(' + '));
```